Tests/BetterCodableTests/LossyDictionaryTests.swift

```diff
@@ -75,4 +75,22 @@ class LossyDictionaryTests: XCTestCase {
         XCTAssertEqual(fixture.stringToInt, ["one": 1, "two": 2, "three": 3])
         XCTAssertEqual(fixture.intToString, [1: "one", 2: "two", 3: "three"])
     }
+
+    func testEncodingLosslessDictionaryRetainsKeys() throws {
+        let decoder = JSONDecoder()
+        decoder.keyDecodingStrategy = .convertFromSnakeCase
+
+        let encoder = JSONEncoder()
+        encoder.keyEncodingStrategy = .convertToSnakeCase
+
+        let fixture = Fixture(
+            stringToInt: ["snake_case": 1],
+            intToString: [:]
+        )
+
+        let reencodedFixture = try decoder.decode(Fixture.self, from: encoder.encode(fixture))
+
+        XCTAssertEqual(reencodedFixture, fixture)
+        XCTAssertEqual(Set(reencodedFixture.stringToInt.keys), ["snake_case"])
```

Comment:
This becomes `snakeCase`. Removing `@LossyDictionary` from `Fixture`'s definition makes the test pass.